### ISSUE TITLE
why seq2seq_rl loss = -0.000000, Is this normal?

### ISSUE THREAD

**Issue by yearsss** [asyml/texar#35]
2018-10-09 09:52:43.044248: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 21534 MB memory) -> physicalGPU (device: 2, name: Tesla P40, pci bus id: 0000:07:00.0, compute capability: 6.1)
2018-10-09 09:52:43.386811: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:3 with 19354 MB memory) -> physicalGPU (device: 3, name: Tesla P40, pci bus id: 0000:08:00.0, compute capability: 6.1)
2018-10-09 09:52:43.724525: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:4 with 21534 MB memory) -> physicalGPU (device: 4, name: Tesla P40, pci bus id: 0000:0c:00.0, compute capability: 6.1)
2018-10-09 09:52:44.066564: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:5 with 3867 MB memory) -> physical GPU (device: 5, name: Tesla P40, pci bus id: 0000:0d:00.0, compute capability: 6.1)
2018-10-09 09:52:44.130317: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:6 with 7327 MB memory) -> physical GPU (device: 6, name: Tesla P40, pci bus id: 0000:0e:00.0, compute capability: 6.1)
2018-10-09 09:52:44.248129: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:7 with 21534 MB memory) -> physicalGPU (device: 7, name: Tesla P40, pci bus id: 0000:0f:00.0, compute capability: 6.1)
step=1, loss=-25.0021, reward=7.1089
step=10, loss=-0.0015, reward=0.0912
step=20, loss=-0.0000, reward=0.0994
step=30, loss=-0.0000, reward=0.0591
step=40, loss=-0.0000, reward=0.0631
step=50, loss=-0.0000, reward=0.0736
step=60, loss=-0.0000, reward=0.0486
step=70, loss=-0.0000, reward=0.0852
step=80, loss=-0.0000, reward=0.1246
step=90, loss=-0.0000, reward=0.0655
step=100, loss=-0.0000, reward=0.1117
step=110, loss=-0.0000, reward=0.1609
step=120, loss=-0.0000, reward=0.1163
step=130, loss=-0.0000, reward=0.0672
step=140, loss=-0.0000, reward=0.0499
